lib/restaurant.rb

```diff
@@ -0,0 +1,13 @@
+class Restaurant
+
+  def menu
+    @menu = [
+      {dish: 'Kebab', price: '£5.00'},
+      {dish: 'Large Fries', price: '£1.50'},
+      {dish: 'Regular Fries', price: '£1.00'},
+      {dish: 'Pounded Yam', price: '£2.00'},
+      {dish: 'Egusi Soup', price: '£4.50'},
+      {dish: 'Jellof Rice', price: '£2.00'},
+    ]
```

Comment:
Have a read about the  [open/closed principle](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle). Can you find a way of moving the menu array outside of the source code of the class?